samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -224,3 +226,12 @@ const videoBackgroundImages = [
     tooltipText: 'Custom Background'
   }
 ];
+
+/* @conditional-compile-remove(teams-adhoc-call) */
+const onFetchProfile = async (userId: string, defaultProfile?: Profile): Promise<Profile | undefined> => {
+  // TODO: Explore using Graph API to differentiate Call queues from other Voice app bots
+  if (userId.startsWith('28:orgid:')) {
```

Comment:
is this always true? I believe all bots use this prefix

Reply:
Don is right, Bots use this prefix.

Reply:
actually, bots use 28: prefix. The orgid is something different. 28:orgid could be specific to call queue as they would technically be the organizer of the meeting. Can we confirm if other bots can be organizers?

Reply:
Oh this is a good point ^ I think if call queues are special and get the orgid moniker like users this might actually be good, is there someone on VoiceApps you can reach out to about this @mgamis-msft?

Reply:
The 28: prefix is actually for resource accounts in Microsoft 365. Each resource account can be assigned to a call queue or an auto attendant. Even Graph API can't know which Voice App a resource account is assigned to. 
![image](https://github.com/Azure/communication-ui-library/assets/79475487/37e75d4a-2056-426f-982b-97d052b497fe).
I will remove this code for our Calling sample but the point of this PR is now Contoso should be able to change the display name of their call queue if they know the resource account user id which they can do using Graph API.